homeassistant/components/automation/blueprints/motion_light.yaml

```diff
@@ -10,7 +10,7 @@ blueprint:
       selector:
         entity:
           filter:
-            device_class: motion
+            device_class: occupancy
```

Comment:
Why would we change the default here just because of a user using occupancy sensors instead of motion sensors?